state/txindex/indexer_service.go

```diff
@@ -99,7 +99,7 @@ func (is *IndexerService) OnStart() error {
 						return
 					}
 				} else {
-					is.Logger.Info("indexed block events", "height", height)
+					is.Logger.Debug("indexed block events", "height", height)
```

Comment:
This one I saw it today quite often in the logs but I didn't really find it useful. I'd agree on "demoting" it to Debug